content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -36,9 +36,9 @@ of **1000** services and **2000** sidecars with 70,000 mesh-wide requests per se
 After running the tests using Istio {{< istio_release_name >}}, we get the following results:
 
 - The Envoy proxy uses **0.5 vCPU** and **50 MB memory** per 1000 requests per second going through the proxy.
-- The `istio-telemetry` service uses **0.6 vCPU** per 1000 **mesh-wide** requests per second.
+- The `istio-telemetry` service uses **0.6 vCPU** per 1000 **mesh-wide** requests per second for Mixer user only.
```

Comment:
I think the phrasing of "for Mixer user only" is a little confusing. I think the point you're trying to convey is that this is the expected usage of istio-telemetry, and it's only relevant if Mixer is in use.

How about "for deployments that use Mixer. This is not relevant for Telemetry V2 deployments" (could link to https://istio.io/docs/reference/config/telemetry/)

Reply:
Good catch! Let me update it.

Reply:
done.